examples/bloopsaphone/working/a_most_magical_instrument.rb

```diff
@@ -0,0 +1,166 @@
+# require 'shoes'
+
+# Shoes.app title: "Cool Soundboard", width: 300, height: 250 do
```

Comment:
kind of a dump of things that worked and trying to iterate more and more difficult